modules/eco-setting-node-to-maintenance-mode-with-cr.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-to-maintenance-mode-with-cr_{context}"]
+= Setting a node to maintenance mode with a NodeMaintenance custom resource
```

Comment:
why chapters "Checking status of current NodeMaintenance CR tasks" and "Resume..." were before "Start..."?

Reply:
@anna-savina I am not sure if i understand the question. Can you please elaborate?

Reply:
I was talking about structure of document, but I saw better [comment](https://github.com/openshift/openshift-docs/pull/42107#pullrequestreview-894042263) from jeana-redhat. So just ignore this my comment, please